certbot-dns-linode/certbot_dns_linode/__init__.py

```diff
@@ -27,14 +27,16 @@
 
 Use of this plugin requires a configuration file containing Linode API
 credentials, obtained from your Linode account's `Applications & API
-Tokens page <https://manager.linode.com/profile/api>`_.
+Tokens page v3 <https://manager.linode.com/profile/api>`_ or `Applications &
+API Tokens page v4 <https://cloud.linode.com/profile/tokens>`_.
```

Comment:
I'm not sure exactly what change to make here, but I think I might find this a bit confusing if I didn't already know what "v3" and "v4" were. A few more words here might be plenty to help new users understand.

Reply:
I'll try to change my wordings on this one.